### PR TITLE
[CodeClean/Service] parse json in rpk

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, define function to parse json file in rpk, including resource and pipeline update.

### CODE REVIEW

File: daemon/pkg-mgr.cc [hunk 1]
```diff
@@ -54,6 +54,92 @@ _get_app_info (const gchar *package_name, const gchar *res_type, const gchar *re
   return json_generator_to_data (gen, NULL);
 }
 
+/**
+ * @brief Parse model json and update ml-service database.
+ */
+static void
+_parse_model_json (const gchar *json_path, const gchar *app_info)
```
Comment (songgot) [inline daemon/pkg-mgr.cc:61]: Is app_info nullable?

Comment (jaeyun-jung) [reply daemon/pkg-mgr.cc:61]: internally, impossible.

Comment (songgot) [reply daemon/pkg-mgr.cc:61]: Thank you!!

File: daemon/pkg-mgr.cc [hunk 2]
```diff
@@ -159,66 +245,16 @@ _pkg_mgr_event_cb (const char *type, const char *package_name,
...
-        db.set_model (name, model, true, description, app_info, &version);
-        _I ("The model with app_info (%s) is registered as version %u", app_info, version);
-      }
-    } catch (const std::exception &e) {
-      _E ("%s", e.what ());
-    }
-
-    db.disconnectDB ();
-
+    g_autofree gchar *pipeline_json
```
Comment (myungjoo) [inline daemon/pkg-mgr.cc:250]: glib has an API for merging paths. Use https://docs.gtk.org/glib/func.build_filename.html

**Review by myungjoo — APPROVED** [review]

**Pull request merged**